src/java/com/unifina/signalpath/blockchain/EthereumToStreamrTypes.java

```diff
@@ -74,4 +76,19 @@ public static Output asOutput(String type, String name, AbstractSignalPathModule
 			return new Output(owner, name, "Object");
 		}
 	}
+
+	public static void convertAndSend(Output output, JsonElement value) {
+		if (output instanceof ListOutput) {
+			ArrayList val = new Gson().fromJson(value, ArrayList.class);
+			output.send(val);
```

Comment:
These lines 80...83 are the gist of this commit that fixes the issue, probably. Everything else is refactoring fluff.

Reply:
So in case there's something terminally wrong with the rest, these lines can and should probably be salvaged.